src/App.js

```diff
@@ -15,7 +15,9 @@ class App extends React.Component {
     render() {
       return (
         <main>
-          <h1>Rancid Tomatillos</h1>
+          <header> 
+            <h1 className="app-title">Rancid Tomatillos</h1>
```

Comment:
This looks great! If we have the chance to add future functionality, we can split this header into its own functional component, perhaps with a search method or a login feature.